end_to_end_tests/custom-templates-golden-record/my_test_api_client/api/__init__.py

```diff
@@ -19,3 +20,7 @@ def default(cls) -> Type[DefaultEndpoints]:
     @classmethod
     def parameters(cls) -> Type[ParametersEndpoints]:
         return ParametersEndpoints
+
+    @classmethod
+    def tag1(cls) -> Type[Tag1Endpoints]:
```

Comment:
Neat! cc @bowenwr @GitOnUp on this feature, of using the tags to define class methods on the client. This is just like what we've done on our wrapper client with the "services".